model/user/db.go

```diff
@@ -336,9 +340,71 @@ func GetOrCreateUser(userId, displayName, email, accessToken, refreshToken strin
 		return nil, errors.Wrapf(err, "decoding user '%s'", userId)
 
 	}
+
+	if err := setSlackInformation(env, u); err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message":       "setting slack information for user",
```

Comment:
"could not set Slack information for user"

Reply:
done